trustlessness.adoc

```diff
@@ -58,24 +58,24 @@ no mechanism exists to make payments over a communications channel
 without a trusted party
 ____
 
-It seems we can't have a decentralized system based on trust, and
+It seems that we can't have a decentralized system based on trust, and
 that's why trustlessness is important in Bitcoin.
 
 To use Bitcoin in a trustless manner, you have to run a
 fully-validating Bitcoin node. Only then will you be able to verify
 that the blocks you receive from others are following the consensus
 rules; for example, that the coin issuance schedule is kept and that no
-double spends occur on the blockchain. If you don't run a full node,
+double-spends occur on the blockchain. If you don't run a full node,
 you outsource verification of Bitcoin blocks to someone else and
 trust them to tell you the truth, which means you're not using Bitcoin
 trustlessly.
 
 David Harding has authored
 https://bitcoin.org/en/bitcoin-core/features/validation[an article on
-the bitcoin.org website] which explains what running a full node, or
-using Bitcoin trustlessly, actually helps you with.
+the bitcoin.org website] explaining how running a full node - or
+using Bitcoin trustlessly - actually helps you.
 
-[quote, David Harding, bitcoin.org website]
+[quote, David Harding, Full Validation on bitcoin.org]
```

Comment:
I'd say this page is from 2015, see its Git history: Harding authored it in 2015, and there have been only very minor changes since.
https://github.com/bitcoin-dot-org/Bitcoin.org/commits/48d5130b42480beab1e2deeeabb8b7e7500bbf3c/en/bitcoin-core/features/validation.md